nilearn/datasets.py

```diff
@@ -764,13 +796,12 @@ def fetch_craddock_2011_atlas(data_dir=None, url=None, resume=True, verbose=1):
     See http://www.nitrc.org/projects/cluster_roi/ for more information
     on this parcellation.
     """
-
     if url is None:
         url = "ftp://www.nitrc.org/home/groups/cluster_roi/htdocs" \
               "/Parcellations/craddock_2011_parcellations.tar.gz"
```

Comment:
I used the year specified on NITRC but @banilo is right when he says that the paper has been published in 2012. Let's ask @GaelVaroquaux to take the final decision ;)